app/javascript/react/components/RestaurantShowContainer.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from 'react'
+import RestaurantShowTile from './RestaurantShowTile'
+
+const RestaurantShowContainer = (props) => {  
+  const [restaurant, setRestaurant] = useState({})
+
+  useEffect(() => {
+    let id = props.match.params.id
+    fetch(`/api/v1/restaurants/${id}`)
+    .then((response) => {
+			if (response.ok) {
+				return response
+			} else {
+				let errorMessage = `${response.status} (${response.statusText})`
+				let error = new Error(errorMessage);
+				throw(error);
+			}
+    })
+    .then((response) => {
+    	return response.json()
+    })
+    .then((body) => {
+      setRestaurant(body)
+    })
+    .catch(error => console.error(`Error in fetch: ${error.message}`))
+  }, [])
+
+	return (
+		<div>
+			<RestaurantShowTile restaurant={restaurant} />
+		</div>
+	)
+}
```

Comment:
woooah some indentation here got thrown off! I would check the indentation format settings in your text editor. In `settings` - `Editor: tab size` set to 2. Also check in the bottom right corner of VS code you'll see `tab size` or `space size` to make sure it says `tab size: 2`

Reply:
Looks normal  in our editors but I think we have some issues with the indentation size in vs code. Will fix first thing tomorrow. Thanks!

Reply:
Just resolved the indentation issue on the show container!